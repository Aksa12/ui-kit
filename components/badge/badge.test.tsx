import React from "react";
import Badge from '../badge';
import renderer from 'react-test-renderer';

describe("Badge", () => {
  it("default", () => {
    expect(renderer
    .create(<Badge>default</Badge>)
    .toJSON()).toMatchSnapshot()
  });

  it("success", () => {
    expect(renderer
    .create(<Badge appearance="success">success</Badge>)
    .toJSON()).toMatchSnapshot()
  });

  it("information", () => {
    expect(renderer
    .create(<Badge appearance="information">information</Badge>)
    .toJSON()).toMatchSnapshot()
  });

  it("warning", () => {
    expect(renderer
    .create(<Badge appearance="warning">warning</Badge>)
    .toJSON()).toMatchSnapshot()
  });

  it("danger", () => {
    expect(renderer
    .create(<Badge appearance="danger">danger</Badge>)
    .toJSON()).toMatchSnapshot()
  });
});
