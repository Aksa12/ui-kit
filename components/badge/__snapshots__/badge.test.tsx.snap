// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge danger 1`] = `
<span
  className="css-1x7fm58"
>
  danger
</span>
`;

exports[`Badge default 1`] = `
<span
  className="css-mx4jeh"
>
  default
</span>
`;

exports[`Badge outline 1`] = `
<span
  className="css-12s43p8"
>
  outline
</span>
`;

exports[`Badge primary 1`] = `
<span
  className="css-25ardn"
>
  primary
</span>
`;

exports[`Badge success 1`] = `
<span
  className="css-1xv1zbc"
>
  success
</span>
`;

exports[`Badge warning 1`] = `
<span
  className="css-r5izn1"
>
  warning
</span>
`;
